lib/Sema/CSBindings.cpp

```diff
@@ -1731,31 +1731,35 @@ void BindingSet::dump(llvm::raw_ostream &out, unsigned indent) const {
       break;
 
     case AllowedBindingKind::Subtypes:
-      out << "(subtypes of) ";
+      return "(subtypes of) " + type.getString(PO);
```

Comment:
Let's add this commit to the other PR that fixes spacing issues and keep this scoped to only disjunction/conjunction changes.